apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,14 @@
+You can add some of your own customizations to the default theme.
+For example, you can utilize `defaultFontStyle` to override the default font style and
+`fonts` to override the default medium font size.
+
+```tsx
+import { loadTheme } from 'office-ui-fabric-react';
+
+loadTheme({
+  defaultFontStyle: { fontFamily: 'Comic Sans MS', fontWeight: 'bold' },
```

Comment:
I wonder if we should try to come up with a more realistic example here, since this is our official doc site? (it was just the first thing I thought of to use in an issue comment since it's mildly amusing)